### PR TITLE
add meterpreter chmod command

### PR DESCRIPTION
**Pull request by timwr**
Quick change to add a native chmod command to meterpreter. I got tired of shelling out in order to chmod stuff and sometimes shell/execve isn't available (e.g on iOS, some SELinux contexts).

## Verification

List the steps needed to make sure this thing works

- [x] Land rapid7/mettle#137
- [x] Get a linux meterpreter session
- [x] `meterpreter> chmod 777 somefile`
- [x] **Verify** `meterpreter> ls -l` shows the updated permissions

### CODE REVIEW

**Review by phra — APPROVED** [review]
n1!

File: lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/fs.rb [hunk 1]
```diff
@@ -398,6 +400,20 @@ def cmd_cp(*args)
   alias :cmd_copy :cmd_cp
   alias :cmd_cp_tabs :cmd_cat_tabs
```
Comment (Green-m) [inline lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/fs.rb]: @timwr 
Maybe `alias :cmd_chmod_tabs :cmd_cat_tabs` for its completion?

**Comment by timwr** [comment]
It works the same as the chmod command normally would (except the input is octal e.g 777, no support for +x etc yet).

File: lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/fs.rb (hunk 1 above)
Comment (Green-m) [reply lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/fs.rb]: Thanks for testing!
Maybe a bug, I would try to find it out.

Comment (Green-m) [reply lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/fs.rb]: @timwr : see https://github.com/rapid7/mettle/issues/138

**Comment by bwatters-r7** [comment]
(Putting testing information here for combined testing of this and https://github.com/rapid7/mettle/pull/137)
# Ubuntu 16.02x64 with linux/x64/meterpreter/reverse_tcp

```
msf5 exploit(multi/handler) > run

[*] Started reverse TCP handler on 192.168.135.111:4567 
[*] Sending stage (816260 bytes) to 192.168.134.187

meterpreter > sysinfo
Computer     : 192.168.134.187
OS           : Ubuntu 16.04 (Linux 4.15.0-34-generic)
Architecture : x64
BuildTuple   : x86_64-linux-musl
Meterpreter  : x64/linux
meterpreter > cp revtcpx64.elf revtcpx64.elf.test
meterpreter > ls revtcp*
File globbing not supported with this session
meterpreter > chmod 777 revtcpx64.elf.test
meterpreter > chmod 777 revtcx64.elf.test
[-] stdapi_fs_chmod: Operation failed: 1
meterpreter >
```
Ubuntu 16.05x86 with mlinux/x86/meterpreter/reverse_tcp
```
msf5 exploit(multi/handler) > run

[*] Started reverse TCP handler on 192.168.135.111:4567 
[*] Sending stage (861480 bytes) to 192.168.132.189

meterpreter > sysinfo
Computer     : 192.168.132.189
OS           : Ubuntu 16.04 (Linux 4.15.0-29-generic)
Architecture : i686
BuildTuple   : i486-linux-musl
Meterpreter  : x86/linux
meterpreter > ls -l
Listing: /home/msfuse
=====================

Mode              Size  Type  Last modified              Name
----              ----  ----  -------------              ----
...
100755/rwxr-xr-x  207   fil   2018-09-19 12:19:33 -0500  revtcpx86.elf

meterpreter > cp revtcpx86.elf revtcpx86.elf.test
meterpreter > ls
Listing: /home/msfuse
=====================

Mode              Size  Type  Last modified              Name
----              ----  ----  -------------              ----
...
100755/rwxr-xr-x  207   fil   2018-09-19 12:19:33 -0500  revtcpx86.elf
100664/rw-rw-r--  207   fil   2018-09-19 06:19:24 -0500  revtcpx86.elf.test

meterpreter > chmod 777 revtcpx86.elf.test
meterpreter > ls
Listing: /home/msfuse
=====================

Mode              Size  Type  Last modified              Name
----              ----  ----  -------------              ----
...
100755/rwxr-xr-x  207   fil   2018-09-19 12:19:33 -0500  revtcpx86.elf
100777/rwxrwxrwx  207   fil   2018-09-19 06:19:24 -0500  revtcpx86.elf.test

meterpreter > 
```

**Comment by bwatters-r7** [comment]
# Release Notes
This adds the chmod command to linux-y meterpreters

**Comment by wvu-r7** [comment]
Next up would be to add this to Python Meterpreter.

**Pull request merged**